en-US/snippets/2018/Firefox100-part1.lang

```diff
@@ -0,0 +1,295 @@
+## NOTE: Reference bug 1463556; Deadline: June 11, 2018
+## NOTE: Guidelines on localizing snippets: https://goo.gl/PluA97.  The tone is QUAC (Quick, Untamed, Alert, Clever). Do your best to make it local and culturally relevant.
+## NOTE: Campaign: Part one of the Firefox 100-snippet journey which is a collection of desktop snippets programmed for new Firefox users. Each part contains four weeks of snippets.
+## NOTE: Limit character length to 180, preferably under 160.
+
+
+## MAX_LENGTH: 180
+# Limit character length to 180, preferably under 160. Please rephrase as needed to fit.
+# Campaign: Firefox Accounts for user without an account
+# Week 1
```

Comment:
This information is not really useful for localizers, let's drop it from the file.

Reply:
OK, remove them all. This will help set for the next few parts..